### DISCUSSION TITLE
not able to add team to the project

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3886] [Q&A]
Since few days I am not able to add team to the project which is under organisation communitygis
I am admin for the org. 
Project : [communitygis](https://app.qfield.cloud/a/communitygis/) / [P025_RS_TCET](https://app.qfield.cloud/a/communitygis/P025_RS_TCET/)
Team @communitygis/TCET

**Comment by jguelat** [comment]
Hi Monika,

I'm having the same problem... I know it's not really helpful, but at least you're not alone :D

**Comment by suricactus** [comment]
The issue has been confirmed and fix will be deployed soon with the new QFieldCloud release.

**Comment by suricactus** [comment]
@MonikaShah @jguelat The fix is deployed now.
